aws/values.yaml

```diff
@@ -316,7 +316,7 @@ webhooksCleanup:
   # -- Create a helm pre-delete hook to cleanup webhooks.
   enabled: true
   # -- `kubectl` image to run commands for deleting webhooks.
-  image: bitnami/kubectl:latest
+  image: ghcr.io/nirmata/kubectl:1.29.0
```

Comment:
I think we need to point to the AWS registry right? Just like we have done for admission-controller etc?

Reply:
Right, done!

Reply:
Right, done!